index.js

```diff
@@ -1,20 +1,27 @@
-const gameStartElement = document.getElementById('start-game')
+const gameStartElement = document.getElementById("start-game");
 
-gameStartElement.addEventListener('click', (event) => {
-    alert('Game is under construction')
-})
+gameStartElement.addEventListener("click", event => {
+  alert("Game is under construction");
+});
+
+// Create gameboard
+
+const canvas = document.getElementById('gameboard');
```

Comment:
please separate your logic. wrap this code in a function that sets up the canvas, create another function that sets up keyboard listeners for controlling the player, and starting the game.